contracts/PoolTogetherEVMBridgeChild.sol

```diff
@@ -14,14 +14,21 @@ contract PoolTogetherEVMBridgeChild is FxBaseChildTunnel {
     /// @notice Emitted when a message is sent from the child chain
     event ReceivedMessagesFromRoot(uint256 indexed stateId, address indexed sender, bytes data);
 
+    /// @notice Emitted when a messages from root fail
+    event MessagesFromRootFailed(uint256 indexed stateId, bytes data, bytes failure);
+
     /// @notice Receives messages from data tunnel and disperses them using MultiSend
     /// @param stateId State update Id
     /// @param sender Address of parent chain sender
     /// @param message Sent from parent chain
     function _processMessageFromRoot(uint256 stateId, address sender, bytes memory message) internal override {
         
-        MultiSend.multiSend(message);
-        emit ReceivedMessagesFromRoot(stateId, sender, message);
+        try this.MultiSend.multiSend(message)  { 
```

Comment:
It seems like this would be useful - at least we would have info on why the stateSender tx's are failing. However try-catch does not work with an internal function call.

I tried adding `this.MultiSend.multisend(message)` and `address(this).MultiSend.multisend(message)` as per https://forum.openzeppelin.com/t/a-brief-analysis-of-the-new-try-catch-functionality-in-solidity-0-6/2564 but neither of these compile (complains it cannot lookup the function).

First: Is this something we definitely want? Second: any idea of why this syntax isnt working?

@asselstine